_digest/october-2020/using-apps-and-software-to-manage-time.mdx

```diff
@@ -0,0 +1,107 @@
+these apps is that many of your small tasks can not be easily managed and kept track of on a calendar, for this reason, using reminder apps is also a great idea. </p>
+
+<div align="center">
+    <figure>
+        <img src="/resources/blog/october-2020/google-calendar.png" alt="Google calendar" width="500"/>
+    </figure>
+</div>
+
+<p> By using reminder apps, you can make a list of tasks or simple reminders that you want to remember. Some of the most used reminder apps are Todoist, 
+Wunderlist, and Toodledo. These apps are all essentially places where you can jot down your to-do list and set reminders for them. By doing this you can stay 
+on top of simple tasks, homework, due dates, etc. Many of these apps also have a feature where you can share your tasks with others for collaboration. 
+These apps are great for the type of people who like to have all of their tasks and work—from tiny things such as watering your plants to big things such as 
+studying for a final exam—written down in one place with reminders being sent to them often on their phone or computer. However, just like calendar apps, these 
+apps aren’t perfect. They generally lack the ability to display when you are planning to do your events. </p>
+
+<div align="center">
+    <figure>
+        <img src="/resources/blog/october-2020/todoist.png" alt="Todoist" width="500"/>
+    </figure>
+</div>
+
+<p> Ultimately, high school students have many things that they need to keep track of and manage. Due to this, students need some sort of app or 
+software that they can use. Most of these apps fall into two types: calendar apps and reminder apps. However, neither of these types can do everything, 
+and for the reasons mentioned in this article, using a combination of calendar apps and reminder apps is the best way to manage your time and tasks. </p>
+
+
+<table>
+  <tr>
+    <th> </th>
+    <th> Calendar Apps </th> 
+    <th> Reminder Apps </th>
+  </tr>
+  <tr>
+    <td> Keep track of Events </td>
+    <td> √ </td> 
+    <td> </td>
+  </tr>
+  <tr>
+    <td> Keep track of time </td>
+    <td> √</td> 
+    <td> </td>
+  </tr>
+  <tr>
+    <td> Keep track of Homework </td>
+    <td> </td> 
+    <td> √ </td>
+  </tr>
+  <tr>
+    <td> Keep track of Tasks </td>
+    <td> </td> 
+    <td> √ </td>
+  </tr>
+  <tr>
+    <td> Sends you notification reminders </td>
+    <td> √ </td> 
+    <td> √ </td>
+  </tr>
+  <tr>
+    <td> Collaborative work </td>
+    <td> √ </td> 
+    <td> √ </td>
+  </tr>
+</table>
+
+<p> If you found this article helpful, you may want to check out the other parts of this series. Here are their links: </p>
+
+<ul>
+    <li> <a href="blog/making-the-most-out-of-your-morning" target="_blank" rel="noopener noreferrer"> Making the most out of your morning: Establishing Morning Habits en route to medical school </a></li>
```

Comment:
remove the `blog/` from each of these. This is causing these links to refer to `/resources/blog/blog/<title>`, which is obviously wrong and just leads to the 404 page.